openrtb_ext/bidders_test.go

```diff
@@ -50,12 +50,11 @@ func TestInvalidParams(t *testing.T) {
 	}
 }
 
-func TestBidderListDoesNotDefineGeneral(t *testing.T) {
-	assert.NotContains(t, CoreBidderNames(), BidderNameGeneral)
-}
-
-func TestBidderListDoesNotDefineContext(t *testing.T) {
-	assert.NotContains(t, CoreBidderNames(), BidderNameContext)
+func TestBidderReservedNames(t *testing.T) {
```

Comment:
This confused me for a while. This is testing that there are no reserved names in the core bidder list, not testing that `IsBidderNameReserved()` is working properly. Perhaps a comment to make this clearer would help?

Reply:
Understood. I'll change it's name and see if that's better.

Reply:
Actually, I also a think a refactoring and proper validation test coverage would help here too.